tests/status/run.sh

```diff
@@ -10,63 +10,61 @@ STATUS_LOG="${OUT_DIR}/status.log"
 
 # use latest ts as initial-commit-ts, so we can skip binlog by previous test case
 args="-initial-commit-ts=-1"
-down_run_sql "DROP DATABASE IF EXISTS tidb_binlog"
+# down_run_sql "DROP DATABASE IF EXISTS tidb_binlog"
```

Comment:
this might cause the running drainer just fatal (failed to update tidb_binlog.checkpoint) which is not expected. 
It looks want:
1. stop the running drainer
2. clean the checkpoint table
3. start the new drainer

but here can't works as expected, it stops the running drainer in `run_drainer` , so I just change to test of `status` to use a different schema for saving checkpoint.